content/docs/ai/prompt-enrichment.md

```diff
@@ -0,0 +1,330 @@
+         }
+      ]
+     }' | jq -r '.choices[].message.content'
+   ```
+   {{< /tab >}}
+
+   {{< /tabs >}}
+
+   Verify that the request succeeds and that you get back a structured CSV response.
+   ```
+   City,Continent
+   Seattle,North America
+   Los Angeles,North America
+   Chicago,North America
+   London,Europe
+   Paris,Europe
+   Berlin,Europe
+   ```
+
+2. Refactor the request to improve readability and management of the prompt. In the following example, the instructions are separated from the unstructured text. The instructions are added as a system prompt and the unstructured text is added as a user prompt.
+
+   {{< tabs items="Cloud Provider LoadBalancer,Port-forward for local testing" >}}
+
+   {{< tab >}}
+   ```sh
+   curl "$INGRESS_GW_ADDRESS:8080/openai" -H content-type:application/json -d '{
+      "model": "gpt-3.5-turbo",
+      "messages": [
+        {
+          "role": "system",
+          "content": "Parse the unstructured text into CSV format."
+        },
+        {
+          "role": "user",
+          "content": "Seattle, Los Angeles, and Chicago are cities in North America. London, Paris, and Berlin are cities in Europe."
+        }
+      ]
+    }' | jq -r '.choices[].message.content'
+   ```
+   {{< /tab >}}
+
+   {{< tab >}}
+   ```sh
+   curl "localhost:8080/openai" -H content-type:application/json -d '{
+      "model": "gpt-3.5-turbo",
+      "messages": [
+        {
+          "role": "system",
+          "content": "Parse the unstructured text into CSV format."
+        },
+        {
+          "role": "user",
+          "content": "Seattle, Los Angeles, and Chicago are cities in North America. London, Paris, and Berlin are cities in Europe."
+        }
+      ]
+    }' | jq -r '.choices[].message.content'
+   ```
+   {{< /tab >}}
+
+   {{< /tabs >}}
+
+   Verify that you get back the same output as in the previous step.
+   ```
+   City, Continent  
+   Seattle, North America  
+   Los Angeles, North America  
+   Chicago, North America  
+   London, Europe  
+   Paris, Europe  
+   Berlin, Europe
+   ```
+
+## Append or prepend prompts
+
+Use a RoutePolicy resource to enrich prompts by appending or prepending system and user prompts to each request. This way, you can centrally manage common prompts that you want to add to each request.
+
+1. Create a RoutePolicy resource to enrich your prompts and configure additional settings. The following example prepends a system prompt of `Parse the unstructured text into CSV format.` to each request that is sent to the `openai` HTTPRoute. Note that this RoutePolicy also disables the 15 second default [Envoy route timeout](https://www.envoyproxy.io/docs/envoy/latest/faq/configuration/timeouts#route-timeouts). This setting is required to prevent timeout errors when sending requests to an LLM. Alternatively, you can also set a timeout that is higher than 15 seconds. 
```

Comment:
i think we are either missing the timeout setting, or if it is not needed anymore then can remove this text